hack/kubevirt-builder/Dockerfile.ci

```diff
@@ -0,0 +1,131 @@
+# python3-pip-9.0.3-5.el7.noarch \
+RUN yum install -y yum-plugin-copr && \
+    yum copr enable -y vbatts/bazel epel-7 && \
+    yum -y install \
+        libvirt-devel \
+        bazel \
+        cpio \
+        patch \
+        make \
+        git \
+        mercurial \
+        sudo \
+        gcc \
+        gcc-c++ \
+        glibc-devel \
+        findutils \
+        qemu-img \
+        protobuf-compiler \
+        python3-devel \
+        python3-pip \
+        redhat-rpm-config \
+        jq \
+        unzip && \
+    yum clean all && rm -rf /var/cache/yum/*
+
+# install gradle
+RUN yum install -y java-1.8.0-openjdk-devel && \
+    wget https://services.gradle.org/distributions/gradle-5.0-bin.zip -P /tmp && \
+    unzip -d /opt/gradle /tmp/gradle-5.0-bin.zip
+
+ENV GIMME_GO_VERSION=1.12.8
+
+RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=/gimme bash >> /etc/profile.d/gimme.sh
+
+ENV GOPATH="/go" GOBIN="/usr/bin"
+
+# Install persistent go packages
+RUN \
+    mkdir -p /go && \
+    source /etc/profile.d/gimme.sh && \
+    # Install goimports
+    go get -u -d golang.org/x/tools/cmd/goimports && \
+    cd $GOPATH/src/golang.org/x/tools/cmd/goimports && \
+    git checkout release-branch.go1.12 && \
+    go install && \
+    # Install mvdan/sh
+    git clone https://github.com/mvdan/sh.git $GOPATH/src/mvdan.cc/sh && \
+    cd $GOPATH/src/mvdan.cc/sh/cmd/shfmt && \
+    git checkout v2.5.0 && \
+    go get mvdan.cc/sh/cmd/shfmt && \
+    go install && \
+    go get -u -d k8s.io/code-generator/cmd/deepcopy-gen && \
+    go get -u -d k8s.io/code-generator/cmd/defaulter-gen && \
+    go get -u -d k8s.io/kube-openapi/cmd/openapi-gen && \
+    go get -u -d github.com/golang/protobuf/protoc-gen-go && \
+    # Install deepcopy-gen
+    cd $GOPATH/src/k8s.io/code-generator/cmd/deepcopy-gen && \
+    git checkout kubernetes-1.13.4 && \
+    go install && \
+    # Install defaulter-gen
+    cd $GOPATH/src/k8s.io/code-generator/cmd/defaulter-gen && \
+    git checkout kubernetes-1.13.4 && \
+    go install && \
+    # Install openapi-gen
+    cd $GOPATH/src/k8s.io/kube-openapi/cmd/openapi-gen && \
+    git checkout c59034cc13d587f5ef4e85ca0ade0c1866ae8e1d && \
+    go install && \
+    # Install protoc-gen-go
+    cd $GOPATH/src/github.com/golang/protobuf/protoc-gen-go && \
+    git checkout 1643683e1b54a9e88ad26d98f81400c8c9d9f4f9 && \
+    go install
+
+
+RUN \
+    source /etc/profile.d/gimme.sh && \
+    go get github.com/mattn/goveralls && \
+    go get -u github.com/golang/mock/gomock && \
+    go get -u github.com/rmohr/mock/mockgen && \
+    go get -u github.com/rmohr/go-swagger-utils/swagger-doc && \
+    go get -u github.com/onsi/ginkgo/ginkgo
+
+RUN pip3 install j2cli && pip3 install operator-courier==1.3.0
+
+WORKDIR /tmp/kubevirt
+COPY . .
+
+# update git
+COPY hack/kubevirt-builder/wandisco-git.repo /etc/yum.repos.d/wandisco-git.repo
```

Comment:
why can't we use default git here?

Reply:
default git is old in Centos 7 (1.8.3.1), so there are build failures when 'git fetch-ing` during `go build`